lib/flash.toit

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2020 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the lib/LICENSE file.
+
+import spi
+import reader show Reader
+import writer show Writer
+import gpio
+
+
+FLASH_5MHZ    ::= 0 ///< The flash runs under 5MHz
```

Comment:
Not sure the comments are helpful. If you want to keep them, I would reformat them:

```suggestion
FLASH_5MHZ    ::= 0 // The flash runs at 5MHz.
```